<?xml version="1.0" encoding="utf-8"?>
<testsuite errors="0" failures="0" name="telemetry.test.test_error" tests="7" time="0.0633356571198"><testcase classname="telemetry.test.test_error.TestUnhandledException" name="test_unhandledexception" time="0.0437650680542"></testcase><testcase classname="telemetry.test.test_error.TestUnhandledExceptionReportGeneration" name="test_generate_exception_report_returns_template" time="0.0150830745697"></testcase><testcase classname="telemetry.test.test_error.TestUnhandledExceptionReportGeneration" name="test_send_exception_report_no_recipient" time="0.00102591514587"></testcase><testcase classname="telemetry.test.test_error.TestUnhandledExceptionReportGeneration" name="test_create_exception_record_query_filter" time="0.000970840454102"></testcase><testcase classname="telemetry.test.test_error.TestUnhandledExceptionReportGeneration" name="test_send_exception_report_valid_recipient" time="0.00137090682983"></testcase><testcase classname="telemetry.test.test_error.TestKeyName" name="test_keyname" time="0.000702857971191"></testcase><testcase classname="telemetry.test.test_error.TestSignature" name="test_signature" time="0.000416994094849"></testcase></testsuite>
